docs/running.rst

```diff
@@ -787,180 +782,24 @@ By default, the output in ``reframe.log`` looks like the following:
+Every time ReFrame is run, the ``reframe.log`` files will be rewritten.
+However, you can ask ReFrame to copy it to the output directory before exiting by passing it the ``--save-log-files`` option.
 
-
-Syslog log handler
-""""""""""""""""""
-
-In addition to the common log handler attributes, file log handlers accept the following:
-
-* ``socktype``: The type of socket where the handler will send log records to. There are two socket types:
-
-   1. ``udp``: (default) This opens a UDP datagram socket.
-   2. ``tcp``: This opens a TCP stream socket.
-
-* ``facility``: (default: ``user``) The Syslog facility to send records to.
-  The list of supported facilities can be found `here <https://docs.python.org/3.6/library/logging.handlers.html#logging.handlers.SysLogHandler.encodePriority>`__.
-* ``address``: (required) The address where the handler will connect to.
-  This can either be of the form ``<host>:<port>`` or simply a path that refers to a Unix domain socket.
-
-
-.. note::
-   .. versionadded:: 2.17
+Users have full control over what is being logged, where it is send and how log records are formatted.
+The `configuration guide <configure.html#logging-configuration>`__ gives an overview of the logging configuration, whereas a complete description of the available options can be found in the `configuration reference <config_reference.html#logging-configuration>`__.
 
 
+-------------------
 Performance Logging
-^^^^^^^^^^^^^^^^^^^
+-------------------
 
-ReFrame supports an additional logging facility for recording performance values, in order to be able to keep historical performance data.
-This is configured by the ``perf_logging_config`` variables, whose syntax is the same as for the ``logging_config``:
-
-.. literalinclude:: ../reframe/settings.py
-  :lines: 76-95
-  :dedent: 4
-
-Performance logging introduces two new log record handlers, specifically designed for this purpose.
-
-File-based Performance Logging
-""""""""""""""""""""""""""""""
-
-The type of this handler is ``filelog`` and logs the performance of a regression test in one or more files.
-The attributes of this handler are the following:
-
-* ``prefix``: This is the directory prefix (usually dynamic) where the performance logs of a test will be stored.
-  This attribute accepts any of the check-specific formatting placeholders described `above <#common-log-handler-attributes>`__.
-  This allows you to create dynamic paths based on the current system, partition and/or programming environment a test executes.
-  This dynamic prefix is appended to the "global" performance log directory prefix, configurable through the ``--perflogdir`` option or the ``perflogdir`` attribute of the `system configuration <configuring.html#system-configuration>`__.
-  The default configuration of ReFrame for performance logging (shown in the previous listing) generates the following files:
+Performance testing is an essential part of ReFrame and performance data from tests is handled specially.
```

Comment:
Do you propose this because you are a "data is plural" proponent? Because there two different independent clauses here, so the "is" refers to the "performance data." I suggest sticking to the singular form of data, since this is what we are using throughout.